ord_schema/interface/ord_client_test.py

```diff
@@ -52,14 +53,53 @@ def test_fetch_reaction(self, reaction_id, created_by):
         'ord-c6fbf2aab30841d198a27068a65a9a98'
     ], ['Jun Li', 'Steven Kearnes']))
     def test_fetch_reactions(self, reaction_ids, created_by):
-        client = ord_client.OrdClient()
-        reactions = client.fetch_reactions(reaction_ids)
+        reactions = self.client.fetch_reactions(reaction_ids)
         self.assertLen(reaction_ids, len(created_by))
         self.assertLen(reactions, len(created_by))
         for reaction, expected in zip(reactions, created_by):
             self.assertEqual(reaction.provenance.record_created.person.name,
                              expected)
 
+    def test_query_reaction_ids(self):
+        dataset = self.client.query(reaction_ids=[
+            'ord-f0621fa47ac74fd59f9da027f6d13fc4',
+            'ord-c6fbf2aab30841d198a27068a65a9a98'
+        ])
+        self.assertLen(dataset.reactions, 2)
+
+    def test_query_reaction_smarts(self):
+        dataset = self.client.query(
+            reaction_smarts='FC(F)(F)c1ccc([F,Cl,Br,I])cc1>CS(=O)C>')
+        self.assertLen(dataset.reactions, 862)
```

Comment:
Are all of these tests making use of test data? Or the live database? If the latter, it seems unwise to pin our tests to specific numbers of results